File: Iniciante/3170.cpp
#include <iostream>

using namespace std;

int main(){
    int b, g;
    cin >> b >> g;
    if (g % 2 == 0){
        if (g/2 <= b){
            cout << "Amelia tem todas bolinhas!" << endl;
        } else {
            cout << "Faltam " << g/2 - b << " bolinha(s)" << endl;
        }
    } else{
        while (g % 2 != 0){
            g -= 1;
        }
        if (g/2 <= b){
            cout << "Amelia tem todas bolinhas!" << endl;
        } else {
            cout << "Faltam " << g/2 - b << " bolinha(s)" << endl;
        }
    }
    return 0;
}

File: Iniciante/3170.py
b = int(input())
g = int(input())
if g % 2 == 0:
    if g/2 <= b:
        print('Amelia tem todas bolinhas!')
    else:
        print(f"Faltam {g/2 - b:.0f} bolinha(s)")
else:
    while g % 2 != 0:
        g -= 1
    if g/2 <= b:
        print('Amelia tem todas bolinhas!')
    else:
        print(f"Faltam {g/2 - b:.0f} bolinha(s)")